x-pack/test/ui_capabilities/common/nav_links_builder.ts

```diff
@@ -13,11 +13,14 @@ export class NavLinksBuilder {
       ...features,
       // management isn't a first-class "feature", but it makes our life easier here to pretend like it is
       management: {
-        navLinkId: 'kibana:stack_management',
+        app: ['kibana:stack_management'],
       },
       // TODO: Temp until navLinkIds fix is merged in
```

Comment:
__question:__ do we still need this comment?

Reply:
Yes, the "fix" in question is actually referencing https://github.com/elastic/kibana/issues/66217